### ISSUE TITLE
Android View Clipping 

### ISSUE THREAD

**Issue by vietduc18020326** [sAleksovski/react-native-android-widget#57]
I am using FlexWidget 

![image](https://github.com/sAleksovski/react-native-android-widget/assets/57988353/dceb418c-0ff1-4388-9e25-47e1f7a3ad93)

The left image is what it is currently doing, and the right is what I'd like.
Does anyone have any solution for this?

**Comment by sAleksovski** [comment]
Can you share a minimal widget that reproduces the left picture?

**Comment by vietduc18020326** [comment]
this is my widget
![Screenshot_1702693760](https://github.com/sAleksovski/react-native-android-widget/assets/57988353/67f54270-82f9-4050-8c29-be5cbfcc0580)

**Comment by sAleksovski** [comment]
I meant the widget code that reproduces the image on the left.

**Comment by vietduc18020326** [comment]
This is my code to view card inside widget

```javascript
<FlexWidget
      style={{
        width: 'match_parent',
        height: 69,
        marginBottom: isLast ? 0 : 4,
        borderRadius: 16,
        backgroundColor: WidgetColor.light,
        flexDirection: 'row',
      }}>
      <FlexWidget
        style={{
          height: 'match_parent',
          flex: 1,
          paddingVertical: 8,
          paddingLeft: 8,
          paddingRight: 3,
          flexGap: 2,
        }}>
        <TextWidget
          text={
            booking?.name + (isLast ? 'chadcb chdavc vc havchd avca d' : '') ||
            'No name'
          }
          maxLines={2}
          truncate={'END'}
          style={{
            fontSize: 12,
            color: WidgetColor.neutral1,
            fontWeight: '500',
            fontStyle: 'normal',
            fontFamily: 'Inter-Medium',
          }}
        />
        <TextWidget
          text={stime + ' - ' + etime}
          style={{
            fontSize: 11,
            color: WidgetColor.neutral2,
            fontWeight: '400',
            fontStyle: 'normal',
            fontFamily: 'Inter-Regular',
          }}
        />
        <TextWidget
          text={resource?.name || 'No name'}
          maxLines={1}
          truncate={'END'}
          style={{
            fontSize: 11,
            color: WidgetColor.neutral2,
            fontWeight: '400',
            fontStyle: 'normal',
            fontFamily: 'Inter-Regular',
          }}
        />
      </FlexWidget>
      <FlexWidget
        style={{
          height: 'match_parent',
          width: 5,
          backgroundColor: '#267CDE',
        }}
      />
    </FlexWidget>
``` 

and i dont know how to clipping the blue view and this is the blue view code 

```javascript
<FlexWidget
        style={{
          height: 'match_parent',
          width: 5,
          backgroundColor: '#267CDE',
        }}
      />
```

**Comment by sAleksovski** [comment]
I just released [`0.10.0`](https://github.com/sAleksovski/react-native-android-widget/releases/tag/v0.10.0) that solves this.

You can use the `overflow: 'hidden'` style prop that can clip the view using its radius.
See also the [docs](https://saleksovski.github.io/react-native-android-widget/docs/public-api/interfaces/FlexWidgetStyle#overflow)
